packages/gatsby/src/utils/get-value-at.ts

```diff
@@ -5,6 +5,7 @@ export function getValueAt(obj: object, selector: string | string[]): any {
 }
 
 function get(obj: object, selectors: string[]): any {
+  if (obj === null) return undefined
```

Comment:
Is targeting just `null` safe? Can `obj` ever be `undefined` for example? Just wondering if checking if loosening the condition to check `obj` is falsy would be correct here (and potentially safer). (I have not much context around inference anymore, so I will defer to your judgement)

Reply:
Good question. We have it typed as `object` in function arguments. That's why I am only checking for `null` (as it is technically an object too).

If it can be something else - then our typings are incorrect. Let me check real quick.

Reply:
Yeah, typings are incorrect indeed. I'll try to fix those too.

Reply:
Fixed those, please re-check. And thanks for noticing this!